app/entities/movie_search_result.rb

```diff
@@ -16,25 +16,18 @@ def initialize(title:, in_db:, tmdb_id:, release_date:, vote_average:, overview:
     @lists = nil
   end
 
-  def self.parse_results(results)
+  def self.build_list(results)
```

Comment:
I didn't feel 100% certain that I knew what a `MovieSearch` was, but I know what a `MovieSearchResult` is. Also, `parse` is a little strange, so I changed it to `build_list` since we're actually returning a list of objects.

I feel like the combo of `MovieSearchResult.build_list` is more clear as to what's actually happening here, but I'll open to better words.